aws/docker-pull.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Intended for use on the initial build of a box. Ansible will update/restart
+# if changed subseqsently.
+
+IMAGES=(
+  jrgm/123done
+  mozilla/fxa-auth-db-mysql
```

Comment:
do we need a way to somehow specify the tags, (i.e picking one of these: https://hub.docker.com/r/mozilla/fxa-content-server/tags/ ), should we do this later or part of this PR?